### PR TITLE
feat(hive): add nosasl support

### PR DESCRIPTION
**Pull request by RonBarabash**

### CODE REVIEW

File: docker/hive/start-hive.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ METASTORE_PORT=${METASTORE_PORT:=9083}
 DEFAULT_FS=${DEFAULT_FS:=file:///}
 DB_TYPE=${DB_TYPE:=mysql}
 USE_ATLAS=${USE_ATLAS:=false}
-HIVE_AUTH=${HIVE_AUTH:=NONE}
```
Comment (lyogev) [inline docker/hive/start-hive.sh:10]: Do you think we should change the default?

Comment (RonBarabash) [reply docker/hive/start-hive.sh:10]: hmmm do u want to pass in terraform instead?

Comment (RonBarabash) [reply docker/hive/start-hive.sh:10]: it means we need to change all tests and pass it as env var

Comment (lyogev) [reply docker/hive/start-hive.sh:10]: Yes

Comment (RonBarabash) [reply docker/hive/start-hive.sh:10]: because the e2e tests are coupled to this, im thinking we should leave it as our default

Comment (RonBarabash) [reply docker/hive/start-hive.sh]: ok ill change it

**Pull request merged**